utils.py

```diff
@@ -553,6 +577,28 @@ def install_dependencies(packages):
             f"The script could not determine your {config.OS_NAME} install's package manager or it is unsupported. Your computer is missing the command(s) {missing_packages}. Please install your distro's package(s) associated with {missing_packages} for {config.OS_NAME}.")
 
 
+def check_bad_packages(packages):
+        bad_packages = []
+        package_list = packages.split()
+
+        if config.PACKAGE_MANAGER_COMMAND_QUERY:
+            bad_packages = query_packages("remove", package_list)
+
+        if config.PACKAGE_MANAGER_COMMAND_INSTALL:
+            message = f"Your {config.OS_NAME} computer may have bad packages installed. To continue, the script will attempt to remove the package(s): {bad_packages} by using ({config.PACKAGE_MANAGER_COMMAND_INSTALL}).\n\nWARNING: Automating the removal of packages could leave your system in an unusable state if the removal also removes system-required dependencies.\n\nProceed?"
```

Comment:
Would it be a little clearer to call the packages "conflicting" packages rather than "bad" when communicating with the user?

Reply:
Agreed. I wanted to think of a better term but didn't consider conflict.